site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -441,7 +441,7 @@ include::docs/v1/tech/events-webhooks/_tenant-webhooks-table.adoc[]
 
 include::docs/v1/tech/shared/_enterprise-edition-blurb.adoc[]
 
-// TODO add screenshots
+image::core-concepts/tenant-configuration-scim-settings.png[Tenant Configuration - SCIM,width=1200, role=shadowed]
 
 Read more about setting up and using SCIM in the link:/docs/v1/tech/core-concepts/scim[full SCIM documentation].
```

Comment:
The table below should be updated to match the table in the UI, and some other minor issues:

For example, 
- `SCIM settings` -> `SCIM server settings`
- `Enabled` (field) -> Update this copy to match other fields in the guide, and `Endpoints` should not be capitalized.
- `Enabled` (field) -> This is not required.
- `Client entity type` and `Server entity type`, can you add some text here to describe what to do if you don't have an option to select from? I.e. they need to navigate to `Entity > Types` and create a new one using the template button client or server.
- `SCIM Client` -> `SCIM client`
- `SCIM Server` -> `SCIM server`
- For lambdas, describe what they do as well, they are invoked, but why? 
- Capitalize `JSON` 
- `EnterpriseUser` -> `Enterprise User` (to match the UI)
- `but by default it will response with the core SCIM` not sure that that means, it needs to be re-worked.

Reply:
Think I got all of these.